src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String string;
+    private int variable;
```

Comment:
What does `variable` mean?

Reply:
It contains variable of type int. In this it contain current year -> variable = LocalDate.now().getYear();

Reply:
I think this variable should be named in something like "currentYear"

Reply:
I renamed this variable